src/utils.js

```diff
@@ -1,8 +1,32 @@
 import F from 'futil'
 import _ from 'lodash/fp'
 
-export const setFilterOnly = x =>
-  F.deepMap(
-    F.when(_.isPlainObject, F.setOn('filterOnly', true)),
-    _.cloneDeep(x)
-  )
+let Tree = F.tree(x => x.children)
+export let setFilterOnly = Tree.transform(x => { x.filterOnly = true })
+
+// Wraps provided node in a new `and` group, using the first node/tree's schema and key
+export let andGroup = (...args) => {
```

Comment:
We should export some generic `utils` like this from contexture in some follow-up since this is also used in other repos.  I'll just make a note of it since it's so low impact.